public/data/myanmar/nso/poverty_st.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
@wadhwamatic do we have a real example that we could add to the app instead of a test dataset?

Reply:
@ericboucher - the short answer is we don't have one yet. but as we're integrating data from another source (mVAM) in the near term which may not have matching boundary files, it's eminent. see my comment on this issue: https://github.com/WFP-VAM/prism-frontend/issues/311#issuecomment-989330140

we also will have unique boundary files per round of data in the 'cadre harmonise' datasets for the upcoming regional deployment for west africa. not easy to see distinction in these two examples, but we know that the boundaries will change in some rounds of data collection:
https://github.com/WFP-VAM/RBD_CHIPC_dataprocessing/blob/main/data/geo/finalized_CHgeofiles/wca_CHIPC_nov2021_current_nov2021_simple.geojson
https://github.com/WFP-VAM/RBD_CHIPC_dataprocessing/blob/main/data/geo/finalized_CHgeofiles/wca_CHIPC_nov2021_projected_jun2022_simple.geojson